data_fixtures/models.py

```diff
@@ -0,0 +1 @@
+# Create your models here.
```

Comment:
Are these files (models, tests, views) necessary for Django to recognize `data_fixtures` as an app? If not, can we remove them?

Reply:
its not necessary. I went back and forth on this a bit. my main reason for leaving it was to serve as a reminder that this a django app but I think the apps.py is probably enough. removed the extra files